package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "name": "broccolini.github.com",
+  "version": "1.0.0",
+  "description": "This is the repo for my personal website, built with [Iso](https://github.com/c8r/iso).",
+  "scripts": {
+    "start": "x0 pages -o",
+    "build": "x0 build pages -d . --static"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/broccolini/broccolini.github.com.git"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/broccolini/broccolini.github.com/issues"
+  },
+  "homepage": "https://github.com/broccolini/broccolini.github.com#readme",
+  "devDependencies": {
+    "@compositor/x0": "^5.0.8",
+    "grid-styled": "^4.1.1",
+    "react": "^16.4.1",
+    "styled-components": "^3.3.2",
+    "styled-system": "^2.3.1"
+  },
+  "x0": {
```

Comment:
You can set [configuration options](https://github.com/c8r/x0#configuration) for x0 here